downstream/modules/automation-hub/proc-push-container.adoc

```diff
@@ -25,8 +25,10 @@ $ podman login -u=[username] -p=[password] [automation-hub-url]
 . Push your container image to your {HubName} container registry:
 +
 -----
-$ podman push [automation-hub-url]/[container image name]
+$ podman push [automation-hub-url]/[container image name] --remove-signatures
 -----
++
+NOTE: The `--remove-signatures` flag is required when pushing registry.redhat.io images, as it prevents image recompression issues that causes the command to fail. 
```

Comment:
NOTE: The `--remove-signatures` flag is required when signed images from registry.redhat.io are pushed to the {HubName} container registry. Push operation re-compresses image layers during the upload. The re-compression is not at all guaranteed to be reproducible and  it is client implementation dependent. This can lead to the image layer digest change and  failed push operation with the error `Error: Copying this image requires changing layer representation, which is not possible (image is signed or the destination specifies a digest)`.